src/firefox/remote.js

```diff
@@ -0,0 +1,102 @@
+/* @flow */
+import {createLogger} from '../util/logger';
+import {WebExtError} from '../errors';
+import defaultFirefoxConnector from 'node-firefox-connect';
+
+const log = createLogger(__filename);
+
+
+export default function connect(
+    port: number = 6000,
+    {connectToFirefox=defaultFirefoxConnector}: Object = {}): Promise {
+  return connectToFirefox(port)
```

Comment:
it could be reasonable to `log.info` a message to show the user when the tool is "connecting" to the Firefox Remote Debugging and when it is "connected".

It is something that doesn't happen to often, and so I think that it is something that could be reasonably logged at `info` level, and it could help the user to detect, and report the issue correctly, where the tool is stuck if it doesn't work correctly.